src/PanelTraits/AutoSet.php

```diff
@@ -11,8 +11,10 @@
     /**
      * For a simple CRUD Panel, there should be no need to add/define the fields.
      * The public columns in the database will be converted to be fields.
+     *
+     * @param bool $filtered
```

Comment:
And what does this filtered thing do? ^^ i.e. where's the docs in the PHP doc for it?

Reply:
see: https://github.com/Laravel-Backpack/CRUD/pull/1169#discussion_r159145843